pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -7,6 +7,46 @@
             "#import": [ "base" ]
         },
 
+        "android": {
+            "#import": [ "any" ]
+        },
+        "android-arm64": {
+	           "#import": [ "any" ]
+        },
+        "android-mips": {
+            "#import": [ "any" ]
+        },
+
+        "android.21": {
+	           "#import": [ "android" ]
+	       },
+        "android.21-arm64": {
+	           "#import": [ "android.21", "android-arm64" ]
+        },
+        "android.21-mips": {
+	           "#import": [ "android.21", "android-mips" ]
+        },
+
+        "android.22": {
```

Comment:
That also sounds reasonable to me. How different / incompatible are the different versions, though? If there's very minimal work to get them working after API level 21, then there's not much harm in adding them now. MIPS on the other hand sounds like a much more niche thing that we might not end up doing anytime soon. Disclaimer: I know little about Android API levels or Android hardware in general.